examples/step-6/doc/intro.dox

```diff
@@ -101,7 +101,54 @@ that a cell can be only refined once more than its neighbors), but that we end
 up with these &ldquo;hanging nodes&rdquo; if we do this.
 
 
-<h3> How to deal with hanging nodes </h3>
+<h3> Why adapatively refined meshes? </h3>
+
+Now that you have seen how these adaptively refined meshes look like,
```

Comment:
:)

Reply:
I've changed this as suggested, though it's not clear to me why this refers to a noun. You've seen how they look like -- that could be "like a mesh that has lines not connected on the other side of the face", or "pretty", or "interesting".

But I trust your instinct :-)

Reply:
This case is less clear than the one with a question. I should find someone who actually understands grammar and get a real explanation for this. Regardless: I believe the present version is more idiomatic so this is fine.